### PR TITLE
Return to Home page button on error page

### PR DESCRIPTION
**Pull request by radekkaluzik**
https://issues.redhat.com/browse/RHCLOUD-15772

### CODE REVIEW

File: src/js/App/ErrorComponent/ErrorComponent.js [hunk 1]
```diff
@@ -71,14 +72,15 @@ const ErrorComponent = (props) => {
...
+              </ChromeLink>
+            )}
             variant="primary"
-            onClick={() => {
-              window.history.back();
-            }}
-          >
-            Return to last page.
-          </Button>
+          ></Button>
```
Comment (Hyperkid123) [inline src/js/App/ErrorComponent/ErrorComponent.js:83]: There is no need for extra closing tah. You can use just one
```suggestion
          />
```

File: src/js/App/ErrorComponent/ErrorComponent.js [hunk 2]
```diff
@@ -71,14 +72,15 @@ const ErrorComponent = (props) => {
           </Flex>
         </EmptyStateBody>
         <EmptyStatePrimary>
+          {/* Try to reload the page or&nbsp; */}
```
Comment (Hyperkid123) [inline src/js/App/ErrorComponent/ErrorComponent.js:75]: Can you delete this comment?

File: src/js/App/ErrorComponent/ErrorComponent.js (hunk 1 above)
Comment (radekkaluzik) [reply src/js/App/ErrorComponent/ErrorComponent.js]: Fixed (tag changed to self-closing)

File: src/js/App/ErrorComponent/ErrorComponent.js (hunk 2 above)
Comment (radekkaluzik) [reply src/js/App/ErrorComponent/ErrorComponent.js]: Removed.

**Review by Hyperkid123 — APPROVED** [review]

**Pull request merged**